src/pygeoprocessing/routing/routing.pyx

```diff
@@ -138,6 +138,18 @@ cdef struct FlowPixelType:
     int last_flow_dir
     double value
 
+cdef struct DecayingValue:
+    double decayed_value  # The value, which will be progressively updated as it decays
+    double min_value  # The minimum value before the Decaying Value should be ignored.
+

+cdef struct WeightedFlowPixelType:

```

Comment:
I noticed the other `struct` definitions have a few comments above to describe them. Should these follow that same convention?

Reply:
4f0e062d